civis_jupyter_notebooks/platform_persistence.py

```diff
@@ -15,25 +16,36 @@
 
 def initialize_notebook_from_platform(notebook_path):
     """ This runs on startup to initialize the notebook """
+    logger.info('Getting URL for notebook file')
     client = get_client()
-    nb = client.notebooks.get(os.environ['PLATFORM_OBJECT_ID'])
+    notebook_model = client.notebooks.get(os.environ['PLATFORM_OBJECT_ID'])
 
-    logger.info('Getting URL for notebook file')
-    r = requests.get(nb.notebook_url)
+    logger.info('Pulling contents of notebook file from S3')
+    r = requests.get(notebook_model.notebook_url)
     if r.status_code != 200:
         raise NotebookManagementError('Failed to pull down notebook file from S3')
+    notebook = nbformat.reads(r.content, nbformat.NO_CONVERT)
+
+    notebook_from_s3_is_new = notebook.get('metadata', {}).get('civis', {}).get('new_notebook', False)
+    if notebook_from_s3_is_new:
+        notebook.metadata.civis.new_notebook = False
```

Comment:
can you just delete the metadata in this case?

Reply:
Sure, although the `civis` namespace will still remain.